go.mod

```diff
@@ -406,3 +406,5 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-system-metrics => github.com/AndersonQ/elastic-agent-system-metrics v0.0.0-20230913114254-ef72b6fa59d9
```

Comment:
As soon as https://github.com/elastic/elastic-agent-system-metrics/pull/96 gets merged and `elastic-agent-system-metrics` `v0.5.0` released, I'll remove the replace and properly upgrade `elastic-agent-system-metrics` to `v0.5.0`.